docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -122,6 +122,18 @@ It is recommended you do a [backup]({{< relref "doc/usage/backup-and-restore.en-
 If you have carried out the installation steps as described above, the binary should
 have the generic name `gitea`. Do not change this, i.e. to include the version number.
 
+### 1. Restarting gitea with systemd (recommended)
+
+As explained before, we recommended to to use service system as systemd. In this case ```systemctl restart gitea``` should be enough.
```

Comment:
Ah sorry, there's one more typo (`to to`) and a wording improvement that I missed in the previous review:

It should read `As explained before, we recommend to use systemd as service manager.`

Reply:
patch provided by @6543 and applied